### DISCUSSION TITLE
Reduce the impact of third-party code - Next js

### DISCUSSION THREAD

**Discussion by gauravverma029** [vercel/next.js#18641] [Help] [answered]
How can we handle these script I am already using defer but not seems working
<img width="603" alt="Screen Shot 2020-11-01 at 23 21 43" src="https://user-images.githubusercontent.com/4488516/97816923-60878400-1c99-11eb-82c8-a29224988adf.png">

**Comment by matiishyn** [comment]
how did you solve this? @gauravverma029

**Reply by gauravverma029** [reply]
Still Trying to find Solution but any anyone finds it please share it.

**Comment by MohtashimAli85** [comment]
Did you solve it ?

**Comment by leerob** [comment, marked as answer]
I'd recommend reading through these docs: https://nextjs.org/docs/going-to-production

In this instance, you can improve performance using `next/script`: https://nextjs.org/docs/basic-features/script

**Reply by MohtashimAli85** [reply]
Thank you for replying, I did use `next/script` and I will check the production docs for sure.
But `next/script` didnt solve my issue  but partytown did.
The website on which I am working: stockstelegraph.com
